src/pd/mod.rs

```diff
@@ -88,12 +88,22 @@ pub trait PdClient: Send + Sync {
                         down_peers: Vec<pdpb::PeerStats>)
                         -> Result<pdpb::RegionHeartbeatResponse>;
 
-    // Ask pd for split, pd will returns the new split region id.
+    // Ask pd for split, pd will return the new split region id.
     fn ask_split(&self, region: metapb::Region) -> Result<pdpb::AskSplitResponse>;
 
+    // Ask PD for merge, PD will return the another region id to merged.
+    fn ask_merge(&self, region: metapb::Region) -> Result<pdpb::AskMergeResponse>;
+
     // Send store statistics regularly.
     fn store_heartbeat(&self, stats: pdpb::StoreStats) -> Result<()>;
 
     // Report pd the split region.
     fn report_split(&self, left: metapb::Region, right: metapb::Region) -> Result<()>;
+
+    // Report PD the region merged.
+    fn report_merge(&self,
```

Comment:
I am going to remove this "report merge" protocol and report a region heartbeat when region merge is committed.